requirements/common_constraints.txt

```diff
@@ -1,8 +1,3 @@
-# This is a temporary solution to override the real common_constraints.txt
```

Comment:
I think we don't want to remove this comment, the related work is still in progress.

Reply:
I think this is a bug in the Makefile. On my system (openSUSE), this line with echo [here](https://github.com/openedx/edx-platform/blob/f1542c0762e86e8acae57a33d403a846c645ff88/Makefile#L120) causes the newline escape sequence to be output as is. Perhaps in the system of whoever committed it, it behaves differently. I googled it a bit and it seems "printf" should be more consistent, or using the "-e" flag. I've added that in a separate commit.